app/boardgame-tournaments/[gameTitleSlug]/_components/Updates/Modal.tsx

```diff
@@ -0,0 +1,46 @@
+import Modal from "react-modal";
+
+type Props = {
+  children: React.ReactNode;
+  isOpen: boolean;
+  closeModal: () => void;
+  closeTimeoutMS?: number;
+  className?: string;
+  buttonClassName?: string;
+};
+
+export default function ModalComponent({
+  children,
+  isOpen,
+  closeModal,
+  closeTimeoutMS,
+}: Props) {
+  const windowWidth = window !== undefined ? window.innerWidth : null;
+  const customStyles = {
+    content: {
+      background: "#1f3838",
+      border: "none",
+      maxWidth: windowWidth && windowWidth > 1024 ? "50%" : "100%",
+      margin: "auto",
+    },
+    overlay: {
+      width: "100%",
+      height: "100%",
+      background: "rgba(0,0,0,1)",
+      zIndex: 1000,
+    },
+  };
+  return (
+    <Modal
+      style={customStyles}
+      closeTimeoutMS={closeTimeoutMS}
+      onRequestClose={closeModal}
+      isOpen={isOpen}
+      shouldCloseOnOverlayClick={true}
+      shouldCloseOnEsc={true}
```

Comment:
```suggestion
      shouldCloseOnOverlayClick
      shouldCloseOnEsc
```

no need to explicitly say ={true}

Reply:
oooh... got it. what is this technique called?

Reply:
not sure haha, since it's not undefined, it results to true :))